SpatialGDK/Source/SpatialGDKEditor/Private/Utils/LaunchConfigEditor.cpp

```diff
@@ -18,25 +18,23 @@ void ULaunchConfigurationEditor::PostInitProperties()
 	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
 
 	LaunchConfiguration = SpatialGDKEditorSettings->LaunchConfigDesc;
-	FillWorkerConfigurationFromCurrentMap(LaunchConfiguration.ServerWorkersMap, LaunchConfiguration.World.Dimensions);
+	FillWorkerConfigurationFromCurrentMap(LaunchConfiguration.ServerWorkerConfig, LaunchConfiguration.World.Dimensions);
 }
 
 void ULaunchConfigurationEditor::OnWorkerTypesChanged()
 {
-	LaunchConfiguration.OnWorkerTypesChanged();
-	for (TPair<FName, FWorkerTypeLaunchSection>& LaunchSection : LaunchConfiguration.ServerWorkersMap)
+	// LaunchConfiguration.OnWorkerTypesChanged();
```

Comment:
I removed FSpatialLaunchConfigDescription::OnWorkerTypesChanged, but I'm not sure whether this level still had value or whether I could remove all places where we care about the worker types changing, since there's only one WorkerType now.